src/index.ts

```diff
@@ -24,6 +24,18 @@ export const plugin: PluginFunction<{}> = async schema => {
         return `${name}: [${impls.map(n => `"${n}"`).join(", ")}],`;
       })}
     };
+    
+    ${addUnionTypes(interfaceImpls)}
   `.toStringWithImports();
+
+  // Also add type unions of the possible types for use in code if desired
```

Comment:
This comment looks dangly?

Reply:
LOL - Yes, not sure how that happened 😆 

👍